default.json

```diff
@@ -41,6 +41,11 @@
       "matchPackageNames": ["python", "node"],
       "matchManagers": ["dockerfile"],
       "enabled": false
+    },
+    {
+      "matchPackageNames": ["fourmat"],
```

Comment:
Makes sense, I guess renovate doesn't treat minor bumps on versions < 1 as major? I thought it did/should, or I'm missing something else...

Reply:
yeah, i thought it would by default, too, but apparently not

![CleanShot 2023-04-03 at 14 17 30](https://user-images.githubusercontent.com/2379650/229593438-1e276485-4129-4a04-8888-9f47a1380f81.png)